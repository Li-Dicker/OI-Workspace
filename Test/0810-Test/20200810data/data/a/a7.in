10
196
93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 93410 
44178 34833 47197 25491 18504 40401 10983 37266 26417 20445 45905 32087 22125 42826 31671 31092 19694 21978 21678 1852 22803 24445 32036 4598 34755 38657 18508 27479 14053 17322 48959 36554 44737 37467 12709 36919 48933 11890 4407 49973 13700 28186 40550 2053 36333 34590 17551 17986 40849 43864 49271 47952 23664 18140 13691 46247 33906 7972 29813 23958 32099 16431 25656 2221 24343 30647 48950 27400 4794 38629 32387 763 12206 49030 24037 19576 7950 33278 17649 41283 26748 17272 47592 49417 14961 4948 48185 49503 27538 5138 990 15314 38773 24819 15186 36125 41638 42174 11767 5409 7828 28722 38754 37769 3653 29041 22231 45863 23915 4261 19230 25311 15009 14464 29423 46769 39480 37436 44164 31848 24653 18988 16206 32101 1359 30602 21046 30523 11467 29470 11574 20094 43911 44788 26949 9097 18088 37727 45156 28027 8719 20820 20580 48912 25992 31146 38917 13318 21375 22734 32347 39298 24925 49702 45811 23828 4678 12571 12569 9049 45589 21706 36987 29931 15850 18396 3223 15997 4462 2501 46627 47625 43066 1443 33802 8462 36265 39579 21623 9937 12734 4110 47915 15866 4584 40926 2248 15145 16013 19565 13880 46729 46430 4588 14229 34068 
197
34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 34278 
6763 28814 35831 26085 16290 22893 11070 44290 8305 16296 29388 17061 3409 44628 45482 20569 27178 40180 14223 40351 22773 6308 19246 48136 25284 45891 23821 34059 21638 8453 12083 32435 422 12810 27583 5762 33504 46916 45130 31480 23152 36385 44561 31522 23251 21573 44334 13171 11297 27456 40053 35521 22350 36262 10527 2119 48586 37698 24887 34917 6763 30888 19032 31771 18405 37997 14768 28403 46610 20368 44166 2332 31744 24904 12604 8674 1005 39477 23016 19334 46878 32443 4831 22429 33226 18717 29947 27483 14497 2394 26836 34478 30705 34593 45185 43092 19441 35009 41990 8426 3293 19384 24511 42972 6613 40412 13646 31229 16769 47252 42686 20008 2974 31989 43143 45843 669 27390 21069 34516 32411 38515 9073 17849 14954 6758 24444 23205 9166 44138 11762 24068 43339 25978 40056 26801 35804 32138 45024 28758 9962 22064 13103 41352 23505 7732 11311 8439 27907 17037 43295 45898 11418 11522 27564 5997 34093 26945 1735 16201 1124 2533 31106 45934 11318 42819 33667 2360 40431 16572 24446 13623 38962 14523 42311 25360 7173 39637 35988 34638 27440 12794 26071 39150 9214 23067 5530 13681 313 5811 35794 16471 38535 25321 19767 44652 47014 
200
52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 52880 
15170 6077 33279 41804 42319 982 19433 42613 46150 25033 4641 364 41667 45931 38782 31773 6249 16337 43798 3892 23941 46338 3799 47990 29907 8710 4072 27829 40492 12688 6949 45222 24628 43743 155 11848 3247 25948 20643 10268 9573 37626 40126 32634 21665 27526 26531 31065 18647 24933 46099 8711 36552 20662 22741 30504 9154 31351 32907 7311 40330 18364 20454 10173 28676 23410 32907 33388 29520 14772 23193 14917 5403 19084 38012 8861 1256 38286 20274 41023 619 24771 12889 31323 17099 46017 34755 49406 27544 44520 8995 31891 5452 18985 44786 40375 39366 43222 47030 18333 821 14880 36745 29848 12309 26513 39715 37175 41138 43902 16922 4946 44616 44667 11782 17103 28733 43944 5017 2146 48905 15648 42009 20026 47254 26455 18248 48758 40540 46806 24997 340 40629 14046 38222 11139 13748 32857 35254 2593 33051 39743 9963 48880 22708 27248 8994 2103 22489 34310 17380 26851 35675 15261 31967 49499 20920 49258 11893 35526 4970 37118 18189 45806 9975 1764 23171 36893 2504 46918 20858 18062 23579 2220 1122 11308 42880 24749 30069 31699 25634 24540 38717 30600 48568 43557 36845 17046 18914 24131 18510 44679 30617 15388 49730 22851 15106 33091 35748 10074 
198
88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 88312 
47200 12973 37379 18152 19866 28945 17046 1823 41533 17998 17548 17827 2688 44643 30019 37853 29272 21593 7117 23131 12969 46274 39307 32633 26836 49920 3787 3561 39396 20941 41577 27152 1023 27910 29410 206 45184 45598 41425 40877 11070 41074 22043 23661 45919 494 8309 43092 48641 14758 13575 11572 8995 5009 24848 30555 3346 6211 40053 46677 45575 32567 39497 539 8974 7032 36859 8389 35137 22946 21082 22588 19674 24187 44564 23613 46862 3716 12620 32984 36494 48270 45841 10440 20590 14736 1606 25445 17427 46987 48178 43027 40009 17397 19684 26993 27361 41302 19143 37471 14697 31588 5816 44498 1574 48638 22118 35558 41125 39801 44085 14341 45354 4486 39192 10665 29934 7467 18986 47220 11936 21478 39665 45651 35585 4017 40205 34369 12847 18591 34288 10475 27740 35634 22261 43452 18977 17472 18603 16347 23642 29010 10048 31251 24647 42328 17885 24971 9550 21756 7106 47537 48360 24127 41524 13793 15515 43037 48378 9847 4507 32980 44827 17356 25766 2563 13979 20387 23344 44759 36892 7271 36254 22649 17876 43640 6545 22383 42363 20872 38502 44430 30907 9099 38206 18278 1092 45678 3686 27157 11568 23355 5889 31641 41551 13909 35420 13719 
199
52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 52223 
49559 41335 37019 48991 42578 18672 14549 28490 11537 12382 47424 24921 24290 33426 4992 8558 27283 14281 38911 44406 5907 2528 44093 27052 15718 30562 6454 43467 221 49706 6677 19605 41772 37884 11327 18037 25292 35842 24251 16974 35280 16901 6286 15060 34114 22125 828 45004 12062 23445 17314 42088 24976 1365 35383 31918 1842 37018 15641 32450 36287 48775 6275 33077 30657 23995 6103 33573 23072 6345 269 2105 28423 28236 16012 10059 4639 42936 37334 12852 25182 48403 21098 35398 31782 41817 1347 39829 25577 39622 10963 13366 7800 34449 27326 22286 7447 45564 16723 30794 5758 48474 21405 18062 45979 30897 1293 38545 20909 49697 5389 11434 39922 32629 37036 10230 20330 30453 48069 37361 9863 25276 35040 39007 28783 40919 36572 17237 49237 13991 29235 32456 1152 24312 32015 41399 36425 3182 41610 13306 25869 5526 42785 32500 46819 30620 6687 40037 5280 17637 530 46948 13506 22977 14931 34186 24226 29210 25616 7261 44143 17239 3132 9638 13562 5270 48483 45246 29610 14066 11651 3669 6260 30080 40402 12960 29873 3081 40244 38994 7172 25476 17930 5164 37401 35163 33957 29645 37802 741 1851 33084 39608 9107 43117 20982 42504 3430 26166 
200
70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 70771 
47990 21656 39399 6519 95 49521 33356 40352 39803 18595 26883 39307 546 24641 27372 16431 27353 1878 12712 1424 37925 30033 33718 45691 2989 27011 46316 31091 25093 26992 49083 6433 1165 47268 9946 89 33185 40778 29868 49858 22838 39895 43419 19062 23757 24109 22114 46374 21117 32022 40415 49863 34565 11929 24664 2728 28778 5425 15292 17646 21295 37687 9134 7993 47213 26932 24838 2594 28998 33286 43370 40027 35087 15592 17608 13002 41703 31533 13870 36517 31100 31018 49073 8405 32082 33312 18175 28843 28806 48211 23679 9911 17297 5277 7971 26806 49699 43009 27284 26732 44548 42159 18121 3190 31825 15270 46788 24166 43804 6847 37240 45788 18451 2274 41893 10796 5419 42758 11237 32930 5978 5292 46092 46086 4915 37277 18471 41424 17352 1296 33609 19691 44758 43199 44199 26894 12117 9903 15901 24216 13272 22430 34729 13064 14642 8801 3459 48690 2964 9165 12947 46996 11757 28324 49519 23695 5642 11576 7302 36609 23043 34157 21487 8287 3874 35984 11049 45538 31242 22187 25401 16190 44346 11632 31634 44158 10474 46073 2516 30994 9290 12458 3057 49953 39698 32767 25810 16822 17527 40505 33507 19196 13083 31855 10361 19359 47540 25968 21168 31130 
198
92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 92662 
15372 12961 15476 28029 43578 37483 2754 8802 27343 29625 13976 23140 1131 38426 28333 38929 21455 25371 34154 29287 44861 31707 5568 23705 48025 42969 29711 7667 49811 4201 23036 48949 43397 46186 18571 9030 19079 4468 20793 36239 31229 24571 19827 10292 25524 22366 30395 27121 6317 14863 26791 3456 25569 39780 10747 32698 49820 36463 42574 2901 26075 27480 3593 14385 19825 12890 9321 6022 36768 2354 38079 49036 6664 5840 17253 17754 45090 33338 41328 49309 42464 13173 12185 47094 1671 6534 26813 25347 17401 20251 4190 17788 37261 17251 44320 17279 7963 8599 48965 26994 29258 14597 12022 2749 4203 35525 17157 29713 6221 30999 16455 13779 12827 47592 16388 34702 16110 35056 14990 39253 2017 39890 3021 3028 5623 46338 37118 32479 14909 25416 47346 11591 15350 46401 44913 45307 2325 2274 5582 43079 49104 25029 36014 16426 30018 22247 7873 34574 13218 21942 5127 29273 34543 11589 48946 26579 12741 32999 19434 20690 8782 36818 32936 23986 40076 39155 2090 7098 15696 27812 35499 7853 38782 3036 20862 35883 33202 26846 33042 32258 39053 29630 20865 20590 21042 1254 24190 12603 35357 14169 12826 48826 10795 11769 45232 5241 5003 18409 
198
44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 44392 
10720 49623 1953 933 44746 5001 37271 7471 27218 5666 19983 40111 13860 35479 3506 49760 46886 1124 5314 39890 20100 19218 12299 25046 15267 13761 23075 35915 28580 38461 32312 21249 21887 45513 28557 42869 8074 22400 15147 18702 46473 41081 26237 29530 46497 7136 6064 33305 32357 4507 47669 41294 12977 42232 30842 39605 5891 28482 30468 48746 18251 6951 37892 26970 48382 28475 15190 12484 1852 10205 36496 48932 34477 11000 36733 48647 47314 4045 45427 33559 21062 23282 2280 42358 47644 31916 41958 47527 23177 13638 36384 49096 30915 40406 19734 42376 40492 1720 414 37842 12236 43352 44346 24480 6694 24506 17623 45509 37993 36666 23615 26482 28103 42754 24222 34535 32672 1487 39376 22714 39797 47939 9368 49827 43714 15843 47790 44123 31578 7433 8648 14211 19124 20220 31752 3045 17892 14121 38480 8842 16722 9248 25391 27342 7887 24733 12074 27959 47290 14136 34080 15958 14332 5302 35227 48293 38351 44396 37496 13711 34347 11569 30150 24467 24208 15744 21396 25066 17998 35812 45105 33580 2065 8787 49659 43168 48983 31060 27144 20319 25233 3427 12194 37045 14642 12741 23203 24587 15436 9267 47754 20197 811 6314 40759 18695 43271 43839 
197
8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 8673 
37154 6661 34052 19353 3412 20952 14737 16018 16403 11143 41622 33445 47767 15388 6607 14598 44220 23020 4567 16242 30988 2227 29627 16969 6358 2841 35887 29941 22398 28804 12637 33672 42953 47834 33352 18559 34357 39805 17488 19780 18229 32230 10161 30008 12600 44519 41515 39843 10846 35200 38450 18184 32814 22329 36105 1937 44876 29431 3360 10178 29590 24107 31224 16993 25053 10196 4486 10441 21564 35681 541 49187 8461 1452 20988 22150 35606 16756 44213 43102 15169 46229 9538 36170 36115 31627 34796 5202 46292 37006 24360 21067 29797 22280 37796 38592 21443 6334 45097 21050 33352 17054 23216 9225 29514 10708 21356 49704 28201 37123 34404 3863 30337 28739 31984 42462 42951 28572 12249 16635 8532 21278 40930 21789 47881 38341 37218 19350 31082 37856 1376 15742 20780 48609 49343 3212 33128 14339 46128 36988 30391 46501 19349 39225 46706 13268 49614 23463 8196 48895 23692 9179 2688 26567 13465 42177 4193 6266 26728 8411 43655 22161 618 26804 10048 20167 658 46857 28673 48058 25323 45605 11524 49910 14217 23858 6355 8611 1412 23156 19795 6581 13607 4055 6335 39282 5780 13457 28411 19357 40844 42194 26772 17044 29717 49242 28801 
197
26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 26746 
14323 32364 29506 4703 44842 40468 45037 11860 32358 36863 20272 42093 23192 46523 41526 3015 8715 5195 6641 21442 24484 24525 33463 11535 11905 24273 29036 19173 4563 7763 16339 33521 972 19969 787 24161 33606 39188 47936 33018 15485 45456 47894 48395 599 8381 10680 8591 30060 31146 20287 26644 45771 23498 29638 3984 44350 38845 4682 48559 14687 12066 737 43668 47099 26948 18659 47897 48672 26790 817 18336 7648 1471 44139 39158 34363 45588 31345 40762 23804 11515 30487 23431 46923 22250 35079 9615 29757 7612 22037 1090 10969 6780 6631 33924 25070 24447 35806 37749 18573 38545 46721 10682 18110 861 15866 22461 28912 2072 43133 18613 3664 13659 33019 30853 46175 31586 48335 29923 6367 47572 48288 24828 37819 24444 22911 33376 16532 27733 25285 16577 26360 25233 33524 20178 26785 22366 12307 5272 42100 6538 2858 14328 13031 45612 34406 419 5503 18845 44515 49145 18910 5395 3710 10199 48969 33409 27440 41765 23505 7953 19431 38290 44058 20428 26312 36576 13284 47822 24787 38069 11122 24963 26668 3179 3202 24730 22495 16714 46262 18300 20798 31194 25974 35661 49259 42847 24711 11400 7408 17826 18334 8326 22651 25747 8109 
