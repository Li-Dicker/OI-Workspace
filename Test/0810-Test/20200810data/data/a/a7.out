Case #1: 22843494
Case #2: 11091383
Case #3: 14889669
Case #4: 21782805
Case #5: 14287578
Case #6: 18403287
Case #7: 22387209
Case #8: 13359101
Case #9: 5896667
Case #10: 9578935
